style.css

```diff
@@ -93,3 +99,320 @@ body {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  flex-grow: 0;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  flex: none;
+  border-radius: 100%;
+}
+
+.role {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 8px;
+  width: 88px;
+  height: 32px;
+  flex: none;
+}
+
+.text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  flex-grow: 0;
+}
+
+.desc {
+  width: 295px;
+  height: 76px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  flex-grow: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+  width: 203px;
+  height: 24px;
+  flex: none;
+  flex-grow: 0;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  flex-grow: 0;
+  list-style-type: none;
+}
+
+.tag a {
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.action {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 295px;
+  height: 72px;
+}
+
+.btn-action {
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  top: calc (50% - 48px/2);
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.see {
+  width: 100px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #396df2;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  padding: 114px 20px;
+  background-color: #fff;
+  width: 375px;
+  border-top-right-radius: 70px;
+}
+
+.myself {
+  width: 327px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.conent {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.action-resume {
+  box-sizing: border-box;
+  margin-top: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 162px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+}
+
+.resume {
+  width: 138px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #396df2;
+}
+
+.skills {
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] Please align this section to the center to be similar to the Figma design and also add the grey line at the bottom (border-bottom) of the Frameworks and Skills as requested by the previous code reviewer.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/213835163-c2b0a7a3-3183-4b8d-af25-c4eba4e37547.png) | ![image](https://user-images.githubusercontent.com/58642949/213835374-50267815-d805-42c9-8cf8-e48b3d5a7842.png) |